README.md

```diff
@@ -1,43 +1,99 @@
-# Mattermost LLM Extensions Plugin
+# Mattermost AI Plugin
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+![Screenshot](/img/mention_bot.png)
+
+## Table of Contents
+
+- [Mattermost AI Plugin](#mattermost-ai-plugin)
+  - [Table of Contents](#table-of-contents)
+  - [Background](#background)
+  - [Install](#install)
+  - [Usage](#usage)
+    - [Conversation](#conversation)
+    - [Thread Summarization](#thread-summarization)
+    - [Answer questions about Threads](#answer-questions-about-threads)
+    - [Chat anywhere](#chat-anywhere)
+    - [React for me](#react-for-me)
+    - [RLHF Feedback Collection](#rlhf-feedback-collection)
+  - [Related Efforts](#related-efforts)
+  - [Contributing](#contributing)
+  - [License](#license)
+
+## Background
+
+**🚀 Check out [our AI developer website](https://mattermost.github.io/mattermost-ai-site/) and join the ["AI Exchange" channel](https://community.mattermost.com/core/channels/ai-exchange) where Mattermost's open source community is sharing AI news and innovation in real time!**
 
 The LLM extensions plugin adds functionality around the use and development of LLMs like GPT-3.5 / 4 and hugging face models within Mattermost. 
 
 Currently at the experimental phase of development. Contributions and suggestions welcome! 
 
-## Conversation
+## Install
+
+1. Clone and enter this repository:
```

Comment:
This really just setts up a MM server. Maybe we should call that out? If you already have one try it there instead. 

I was going to say that we could just link to the server README for this but it seems we removed that at some point. There is still a dead link in the TOC: https://github.com/mattermost/mattermost-server#table-of-contents

Reply:
I added a note in the install header to clarify!